packages/core/src/config/minimal.ts

```diff
@@ -47,6 +47,10 @@ export default {
     'no-servers-empty-enum': 'error',
   },
   oas3_1Rules: {
+    'no-invalid-media-type-examples': {
+      severity: 'warn',
+      allowAdditionalProperties: false,
+    },
```

Comment:
```suggestion
    'no-invalid-media-type-examples': 'warn',
```
Won't `allowAdditionalProperties` be `false` by default?

Reply:
Yes, it is false by default, but do we need to turn it off in mininal? @adamaltman